src/LEDTest/led.cpp

```diff
@@ -1,36 +1,37 @@
-
 #include <Arduino.h>
 
-
-
-
+#define LED_PIN 13 
 
 void setup() {                
   // initialize the digital pin as an output.
   // Pin 13 has an LED connected on most Arduino boards:
-  pinMode(13, OUTPUT);   
+  pinMode(LED_PIN, OUTPUT);   
   Serial.begin(9600);
   Serial.println("  Lets test the bluetooth module");
   Serial.println("Press 1 to light the LED, and 2 to turn it off");
-  Serial.println("Entery: ");  
-   digitalWrite(13, HIGH);
+  Serial.println("Entry: ");  
+  digitalWrite(LED_PIN, HIGH);
```

Comment:
I don't want a variable for the pin, but using `#define` makes it more legible and easier to change if we change the pin.

Reply:
Ok